File: src/main/java/com/sf_lolitahag/action/motion/IMotion.java
package com.sf_lolitahag.action.motion;

import java.awt.Image;

public interface IMotion {

  int getAxisX();

  int getAxisY();

  int getAxisShadowX();

  int getAxisShadowY();

  Image getBodyImage();

  Image getShadowImage();

  boolean isShow();
}

File: src/main/kotlin/com/sf_lolitahag/motion/IMotion.kt
package com.sf_lolitahag.motion

import java.awt.Image

interface IMotion {

    val axisX: Int

    val axisY: Int

    val axisShadowX: Int

    val axisShadowY: Int

    val bodyImage: Image?

    val shadowImage: Image?

    val isShow: Boolean
}
